src/locales/ja.yaml

```diff
@@ -16,9 +16,9 @@ app:
       not_loaded: メッシュを読込しない
     tooltip:
       calibrate: 新しいキャリブレーションを開始し、プロファイル'default'として保存します
-      delete: プロファイルを削除します
+      delete: プロファイルを削除します。これにより、プリンタが再起動します
       load: プロファイルを読込み
-      save: キャリブレーションされたプロファイルをprinter.cfgに書込みします
+      save: キャリブレーションされたプロファイルをprinter.cfgに書込みします。これにより、プリンタが再起動します
```

Comment:
The original in the en.yaml is "Commits calibrated profile to printer.cfg", this doesn't seem correct given that.